.drone.yml

```diff
@@ -90,6 +82,14 @@ pipeline:
     commands:
       - npm run snapshot
 
+  # a11y:
```

Comment:
why do we comment out stuff?

Reply:
since a11y wasn't included until now it gives a ton of errors related to accessibility (mostly related to text/button contrast, missing label in inputs etc). This PR is big as it is and this should be fixed in another task/PR. If it is left uncommented it will break drone build.

Reply:
Do you want to make a separate pr which just adds this?
- I don't think this a11y step needs to be part of this pr

Reply:
This has been already added trough some other PR, I got it when I pulled master (before pushing)

Reply:
@markoarthofer22 could you look at fixing these - we should ensure the a11y step is working
I'm happy to pair if you'd like